### PR TITLE
Add a new contract list type for getting non-added contracts

### PR DESCRIPTION
**Pull request by jcarpanelli**
Fixes https://github.com/zeppelinos/zos/issues/899. I added a new contract list type (`notAdded`) and also renamed the already defined types.

### CODE REVIEW

File: packages/cli/test/prompts/prompt.test.js [hunk 1]
```diff
@@ -119,46 +118,61 @@ describe('prompt', function() {
 
     describe('#contractsList', function() {
       beforeEach('set stub and initialize', function() {
-        sinon.stub(Truffle, 'getContractNames').returns(['Foo', 'Bar']);
-        sinon.stub(ZosPackageFile.prototype, 'dependencies').get(() => ({'mock-stdlib': '1.1.0'}));
+        sinon.stub(Truffle, 'getContractNames').returns(['Foo', 'Bar', 'Buz']);
```
Comment (spalladino) [inline packages/cli/test/prompts/prompt.test.js]: It's `Baz`, not `Buz`!

File: packages/cli/src/prompts/prompt.ts [hunk 2]
```diff
@@ -114,20 +115,22 @@ export function proxiesList(pickProxyBy: string, network: string, packageFile?:
...
+    .keys(localPackageFile.contracts)
+    .map(alias => ({ name: localPackageFile.contracts[alias], alias }))
+    .map(({ name: contractName, alias }) => {
+      const label = contractName === alias ? alias : `${alias}[${contractName}]`;
+      return { name: label, value: alias };
+    });
 
   // get contracts from `build/contracts`
-  if (!source || source === 'fromBuildDir') {
+  if (!source || source === 'built') {
```
Comment (spalladino) [inline packages/cli/src/prompts/prompt.ts]: Is this option used at all? If not, can we remove it?

**Comment by spalladino** [comment]
Looks good, merging! Left one minor comment which can be addressed later.

File: packages/cli/src/prompts/prompt.ts (hunk 2 above)
Comment (jcarpanelli) [reply packages/cli/src/prompts/prompt.ts]: Mmh, I believe we are not using it! My bad!

**Pull request merged**